src/code.sh

```
@@ -81,150 +53,342 @@ _trim_fastq_endings () {
+       # remove header line from STAR-Fusion predicted fusions for Docker
+       sr_predictions_name=$(find /home/dnanexus/in/sr_predictions -type f -printf "%f\n")
+       prefix=$(echo "$sr_predictions_name" | cut -d '.' -f 1)
+
+       # running tests to sense-check R1/R2 arrays and file prefixes"
+       _sense_check_fastq_arrays
+
+       mark-section "Setting subjobs to run FusionInspector"
+
+       # variables to input into the app
+       prefix=$(echo "$sr_predictions_name" | cut -d '.' -f 1)
+       known_fusion_file="${known_fusions_name[@]}"
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)